curvefs/src/client/fuse_client.h

```diff
@@ -348,8 +348,8 @@ class FuseClient {
 
     virtual void FlushData() = 0;
 
-    CURVEFS_ERROR UpdateParentInodeMCTimeAndInvalidNlink(
-        fuse_ino_t parent, FsFileType type);
+    CURVEFS_ERROR UpdateParentMCTimeAndNlink(
+        fuse_ino_t parent, FsFileType type, int32_t nlink);
```

Comment:
As for the third argument `nlink`, I only saw you passed either `1` or `-1`, does other values exist, like 2/3 or -2/-3? And if not, a proper way is using an enum to represent 1 or -1.

Reply:
fixed

Reply:
fixed